capplets/time-admin/data/mate-time-admin.desktop.in

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=Time And Date Manager
+Name[zh_CN]=时间和日期管理
+Comment=Manage Time And Date 
```

Comment:
Don't use Title Case for Comment in mate-time-admin.desktop.in:
```shell
$ git grep "Comment="
capplets/about-me/mate-about-me.desktop.in:_Comment=Set your personal information
capplets/accessibility/at-properties/mate-at-properties.desktop.in:_Comment=Choose which accessibility features to enable when you log in
capplets/appearance/data/mate-appearance-properties.desktop.in:_Comment=Customize the look of the desktop
capplets/appearance/data/mate-theme-installer.desktop.in:_Comment=Installs themes packages for various parts of the desktop
capplets/appearance/theme-save.c:               "Comment=%s\n"
capplets/default-applications/mate-default-applications-properties.desktop.in:_Comment=Select your default applications
capplets/display/mate-display-properties.desktop.in:_Comment=Change resolution and position of monitors and projectors
capplets/keybindings/mate-keybinding.desktop.in:_Comment=Assign shortcut keys to commands
capplets/keyboard/mate-keyboard.desktop.in:_Comment=Set your keyboard preferences
capplets/mouse/mate-settings-mouse.desktop.in:_Comment=Set your mouse preferences
capplets/network/mate-network-properties.desktop.in:_Comment=Set your network proxy preferences
capplets/time-admin/data/mate-time-admin.desktop.in:Comment=Manage Time And Date 
capplets/windows/mate-window-properties.desktop.in:_Comment=Set your window properties
font-viewer/mate-font-viewer.desktop.in:_Comment=View fonts on your system
shell/matecc.desktop.in:_Comment=Configure MATE settings
```
Name and Comment should start with underscore for l10n. Test:
```shell
$ ./autogen.sh --prefix=/usr
$ ./makepot
$ grep "Time And Date Manager" mate-control-center.pot
(changes)
$ ./makepot
$ grep "Time And Date Manager" mate-control-center.pot
```

Remove trailing space in Comment.


*.desktop.in files shouldn't include Name[LOCALE]=TRANSLATION, Comment[LOCALE]=TRANSLATION

Reply:
Has been modified

Reply:
- Don't use Title Case for Comment: Unlike Name which is a title, Comment is a sentence, for that reason you can't use Title Case for Comment.

See other _Comment= lines:
```shell
$ git grep "_Comment="
capplets/about-me/mate-about-me.desktop.in:_Comment=Set your personal information
capplets/accessibility/at-properties/mate-at-properties.desktop.in:_Comment=Choose which accessibility features to enable when you log in
capplets/appearance/data/mate-appearance-properties.desktop.in:_Comment=Customize the look of the desktop
capplets/appearance/data/mate-theme-installer.desktop.in:_Comment=Installs themes packages for various parts of the desktop
capplets/default-applications/mate-default-applications-properties.desktop.in:_Comment=Select your default applications
capplets/display/mate-display-properties.desktop.in:_Comment=Change resolution and position of monitors and projectors
capplets/keybindings/mate-keybinding.desktop.in:_Comment=Assign shortcut keys to commands
capplets/keyboard/mate-keyboard.desktop.in:_Comment=Set your keyboard preferences
capplets/mouse/mate-settings-mouse.desktop.in:_Comment=Set your mouse preferences
capplets/network/mate-network-properties.desktop.in:_Comment=Set your network proxy preferences
capplets/time-admin/data/mate-time-admin.desktop.in:_Comment=Manage Time And Date 
capplets/windows/mate-window-properties.desktop.in:_Comment=Set your window properties
font-viewer/mate-font-viewer.desktop.in:_Comment=View fonts on your system
shell/matecc.desktop.in:_Comment=Configure MATE settings
```

- Remove trailing space in Comment.

Test:
```shell
$  ./autogen.sh --prefix=/usr && ./makepot 
$  grep -A 1 mate-time-admin mate-control-center.pot
#: ../capplets/time-admin/data/mate-time-admin.desktop.in.h:1
msgid "Time And Date Manager"
--
#: ../capplets/time-admin/data/mate-time-admin.desktop.in.h:2
msgid "Manage Time And Date "
```

Reply:
Thank you for your advice.